frontend/src/components/Home.js

```diff
@@ -185,6 +193,17 @@ class Home extends React.Component {
                   resultClass="search-results"
                 />
               </div>
+              {this.state.ready && <Graph key={this.state.tract_id} tract_id={this.state.tract_id}></Graph>}
+
+              {this.state.tractSelected && (
+                <div className="tractDetails">
+                  <h2>{this.state.currentTract.id}</h2>
+                  <h1>Tract name</h1>
+                  <h2>{this.state.currentTract.name}</h2>
+                  <h1>Latest Censes Response Rate</h1>
+                  <h2>{this.state.tractData[this.state.currentTract.id] * 100}%</h2>
+                </div>)}
```

Comment:
was this mostly for you guys to see what information was in the tract? @pkgamma is also adding a component to show detail of the tract that's being hovered over, so we might have to figure out merge conflicts here.